docs/ELK-Stack.md

```diff
@@ -0,0 +1,368 @@
+# ELK Stack
+
+If you wish to run IntelMQ with ELK (Elasticsearch, Logstash, Kibana) it is entirely possible. This guide assumes the reader is familiar with basic configuration of ELK and does not aim to cover using ELK in general. It is based on the version 6.8.0 (ELK is a fast moving train therefore things might change). Assuming you have IntelMQ (and Redis) installation in place, lets dive in.
+
+## Configuring IntelMQ for Logstash
+
+In order to pass IntelMQ events to Logstash we are going to need an output queue, particulary orphaned queue (which simply means there is no IntelMQ bot processing events from this queue). However because IntelMQ system does not like to have an orphaned queue (and the IntelMQ Manager does not allow to create one), we will simply add a new bot at the end of the pipeline (or wherever you see fit) and set it's configuration `enabled` to `false`. The bot type does not matter as it's only a forever disabled dummy bot (it's recommended to choose an output bot, e.g. Rest API Output Bot).
```

Comment:
We dont need an orphaned queue. We have a redis output bot, which can be used here. Please see this https://github.com/certtools/intelmq/tree/develop/intelmq/bots/outputs/redis 
Using a redis output bot avoids all configuration warnings/errors and gives you the ability to have a distributed setup where you dont use shared redis.

Reply:
Thank you, I will look into it, definitely looks like a much cleaner solution.